commands/user.go

```diff
@@ -11,8 +11,7 @@ import (
 
 	"github.com/hashicorp/go-multierror"
 
-	"github.com/mattermost/mattermost-server/v6/model"
-
+  "github.com/mattermost/mattermost-server/v6/model"
```

Comment:
The linter complains that the imports are not in the right order.

Reply:
done